internal/legacy/signals/signals_windows_test.go

```diff
@@ -0,0 +1,165 @@
+//go:build windows
+// +build windows
+
+// Note: this build on windows systems
+
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package signals_test
+
+import (
+	"fmt"
+	"os"
+	"sync"
+	"syscall"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"sigs.k8s.io/promo-tools/v4/internal/legacy/signals"
+)
+
+func TestLogSignal(t *testing.T) {
+	// Define what a test looks like.
+	type sigTest struct {
+		signal    os.Signal // Incoming signal to observe.
+		expected  []string  // Expected logs to be produced.
+		terminate bool      // Determine if signals.LogSignals() should return.
+	}
+	// Create multiple tests.
+	// NOTE: We are unable to observe SIGKILL or SIGSTOP, therefore we will not
+	// test with these syscalls.
+	// Another NOTE by @H0llyW00dzZ: SIGIO,SIGSYS,SIGTTOU,SIGCHLD is not available on Windows
+	// System, so we will not test with these syscalls.
+	sigTests := []sigTest{
+		// {
+		// 	signal: syscall.SIGIO,
+		// 	expected: []string{
+		// 		fmt.Sprintf("Encoutered signal: %s", syscall.SIGIO.String()),
+		// 	},
+		// 	terminate: false,
+		// },
```

Comment:
I think we should probably completely remove this case, not just comment it

Reply:
done